Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -59,7 +62,43 @@ class DisplayHymnViewModel: ObservableObject {
                     self.historyStore.storeRecentSong(hymnToStore: self.identifier, songTitle: title)
             }).store(in: &disposables)
     }
-
+    
+    func fetchHymnChords() {
```

Comment:
could you unit test this method :D

Reply:
I wanted to check with you if we should try and combine this method into fetchHymn? What do you think would be better? If we leave it as is I will go forward with trying to write the unit tests?

Reply:
Hmmm, good question. If I were doing it, I probably would have combined them. But now that I look at it, I think I rather like having two separate methods. It keeps things a lot cleaner. The only thing concern would be how it loads on the simulator/device.

Can you experiment a little bit? I tried turning the database off by changing this line to nil: (https://github.com/HymnalDreamTeam/Hymns-iOS/blob/master/Hymns/Infrastructure/Data/HymnDataStore.swift#L48) and looked at how it loaded.

It looks a little bit janky. The title/lyrics would load, then the lyrics bar would suddenly appear. If that's the case, it might be better to just combine them into one method

Reply:
Just a tip for when you do this: feel free to get rid of the "map" step and do everything in the sink step. I don't think it makes sense to map the hymn to an individual component anymore, since we need multiple things from it.

Reply:
Okay, now I have added prefetching into the fetch hymn chords method. At this point, because it is prefetching and loading the webviews I think it makes more sense to have them as two methods. We definitely don't want the preloading of chords slowing down the other method?

Reply:
I don't think prefetching the sheet music would slow down the hymn fetching at all. We aren't waiting on the prefetch to complete before displaying the lyrics. It should be something like this:

getHymn() -> receive hymn -> prefetch sheet music (asynchronously fire and forget that returns immediately) -> display lyrics

Reply:
Check my recent commit... Hope I did this right. It seems to be working well for me..